unified-runtime/test/adapters/cuda/CMakeLists.txt

```diff
@@ -3,6 +3,12 @@
 # See LICENSE.TXT
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+if(NOT UR_DPCXX)
+    message(WARNING
+        "UR_DPCXX is not defined, skipping some adapter tests for cuda")
```

Comment:
According to the [readme](https://github.com/jchlanda/llvm/blob/stefanatwork/ptx_out/unified-runtime/README.md#L143) this is used to build CTS device binaries, I'd prefer if we said which tests are going to be skipped If we are making an effort to issue a diagnostic, why not make it a meaningful one?

Also on even more nit'esque note: s/cuda/Cuda and s/hip/HIP below.

Reply:
I've changed the wording to state "skipping adapter-specific tests for Cuda/HIP", which should hopefully make it clear that it's the non-conformance adapter tests that are skipped.

I've also updated the readme to say just "testing" rather than "CTS testing". I know it's more ambiguous, but hopefully the warnings at configure time should be sufficient.